src/pages/progress/[id].tsx

```diff
@@ -0,0 +1,17 @@
+import { useRouter } from 'next/router';
+import PageNotFound from '../404';
+import ProgressLayout from '@/components/ProgressForm/ProgressLayout';
+
+const ProgressUpdatesPage = () => {
+    const router = useRouter();
+
+    const { id, dev } = router.query;
+
+    if (!id || dev !== 'true') {
```

Comment:
can you explain what is this condition for ?

Reply:
the feature is behind the feature flag dev=true I am checking if that is passed at the same time I am also checking if the id is present in the URL, if any of this is absent we show a 404 page